Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1418,6 +1665,239 @@ def run_script_command():
+    else:
+        return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def status_command():
+    args = demisto.args()
+    request_id = args.get('request_id')
+    sequence_id = args.get('sequence_id')
+    scope = args.get('scope', 'read')
+
+    sequence_id = None if sequence_id is None else int(sequence_id)
+
+    if scope == 'read':
+        response = status_read_cmd(request_id, sequence_id)
+    elif scope == 'write':
+        response = status_write_cmd(request_id, sequence_id)
+    else:  # scope = admin
+        response = status_admin_cmd(request_id, sequence_id)
+
+    resources: dict = response.get('resources', [])
+
+    output = []
+
+    for resource in resources:
+        errors = resource.get('errors', [])
+        if errors:
+            error_message = errors[0].get('message', '')
+            if not error_message:
+                error_message = f'Could not run command\n{errors}'
+            return_error(error_message)
+
+        sequence_id = int(resource.get('sequence_id',0))
+        output.append({
+            'Complete': resource.get('complete') or False,
+            'Stdout': resource.get('stdout'),
+            'Stderr': resource.get('stderr'),
+            'BaseCommand': resource.get('base_command'),
+            'TaskID': resource.get('task_id'),
+            'SequenceID': sequence_id,
+            'NextSequenceID': sequence_id + 1
+        })
+
+    human_readable = tableToMarkdown(f'Command status results', output, removeNull=True)
+    entry_context = {
+        'CrowdStrike.Command(val.TaskID === obj.TaskID)': output
+    }
+
+    return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def get_extracted_file_command():
+    args = demisto.args()
+    host_id = args.get('host_id')
+    sha256 = args.get('sha256')
+    filename = args.get('filename')
+
+    response = get_extracted_file(host_id, sha256, filename)
+
+    # save an extracted file
+    content_type = response.headers.get('Content-Type','').lower()
+    if content_type == 'application/x-7z-compressed':
+        content_disposition = response.headers.get('Content-Disposition','').lower()
+        if content_disposition:
+            filename = email.message_from_string(f'Content-Disposition: {content_disposition}\n\n').get_filename()
+
+        if not filename:
+            sha256 = sha256 or hashlib.sha256(response.content).hexdigest()
+            filename = sha256.lower() + '.dat'
+
+        return fileResult(filename, response.content)
+
+    return_error('A extracted file is missing in the response')
+
+
+def list_host_files_command():
+    args = demisto.args()
+    host_id = args.get('host_id')
+
+    response = list_host_files(host_id)
+    resources: dict = response.get('resources', [])
+
+    files_output = []
+    file_standard_context = []
+    command_output = []
+
+    for resource in resources:
```

Comment:
need to improve output for no results
![image](https://user-images.githubusercontent.com/31018228/85224423-f39edc00-b3d2-11ea-9a52-2abfd9312371.png)

should be something `No results found`